### ISSUE TITLE
Can we please create a build(more this)/deploy(less this for now) on appveyor?

### ISSUE THREAD

**Issue by fir3pho3nixx** [castleproject/Windsor#189]
Hi Guys

Happy to help walk to us over to core, unfortunately don't have visibility of your builds. Do you guys think we could sort this? 

AppVeyor is mega simples. I have implemented it here: https://github.com/cryosharp/fluentwindsor/blob/master/appveyor.yml

If you think I have checked in my passwords, you are wrong :) 

Gav

**Comment by jonorossi** [comment]
I assume you are aware of our public build server (also supports guest login if you don't want to bother creating an account) which has both Windows and Linux build agents: http://builds.castleproject.org/

For Winsdor we had .NET 3.5, 4.0, 4.0 CP, 4.5 and Packaging build configurations, however I've just got rid of everything except 4.5 and Pack after merging your pull requests. We used to have a Mono one, however that got removed years ago when it wasn't maintained, we've got a Mono one for Castle Core. Especially that we are reducing the number of configurations we could merge them into one and enable builds for pull requests including the build status notification into GitHub, TeamCity has this support.

If we want to support Mono we should do it using the .NET 4.5 build and test on Mono, we should get rid of those really old `MONO26`/`MONO28` configurations in the VS projects.
